.changeset/green-ants-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+"@google-labs/breadboard-schema": minor
```

Comment:
I think this might be major, because the JSON schema could have breaking changes in it. (I sort of regret jumping to 1.0.0 already now, since I think we'll have a lot of breaking changes, but since we'll move to the @breadboard-ai NPM org at some point, it doesn't really matter, we could reset to 0).